charts/rancher-logging/0.4.0/Chart.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+description: Rancher logging helm chart to support logging function in rancher
+icon: https://raw.githubusercontent.com/fluent/fluentd-docs/master/public/logo/Fluentd_square.png
+name: rancher-logging
+version: 0.4.0
```

Comment:
Curious why we went with 0.4.0, was that because 1.22 brought breaking changes ? Not a dealbreaker.

Reply:
https://github.com/rancher/system-charts/pull/552